script/lazytox.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env python
+"""
+Lazy 'tox' to quickly check if branch is up to PR standards.
+
+This is NOT a tox replacement, only a quick check during development.
+"""
+import os
+import asyncio
+import sys
+import re
+import shlex
+from collections import namedtuple
+
+try:
+    from colorlog.escape_codes import parse_colors
+except ImportError:
+    def parse_colors(_):
+        """Empty parse_colors function."""
+        return ''
+
+
+RE_ASCII = re.compile(r"\033\[[^m]*m")
+Error = namedtuple('ERROR',
+                   "file line col msg")  # pylint: disable=invalid-name
+PASS = 'green'
+FAIL = 'bold_red'
+
+
+def printc(the_color, *args):
+    """Color print helper."""
+    msg = ' '.join(args)
+    try:
+        print(parse_colors(the_color) + msg + parse_colors('reset'))
+    except KeyError:
+        print(msg)
+        raise ValueError("Invalid color {}".format(the_color))
+
+
+def validate_requirements_ok():
+    """Validate requirements, returns True of ok."""
+    # pylint: disable=E0402
+    from gen_requirements_all import main as req_main
+    return req_main(True) == 0
+
+
+async def read_stream(stream, display):
+    """Read from stream line by line until EOF, display, and capture lines."""
+    output = []
+    while True:
+        line = await stream.readline()
+        if not line:
+            break
+        output.append(line)
+        display(line.decode())  # assume it doesn't block
+    return b''.join(output)
+
+
+async def async_exec(*args, display=False):
+    """Execute, return code & log."""
+    argsp = []
+    for arg in args:
+        if os.path.isfile(arg):
+            argsp.append("\\\n  {}".format(shlex.quote(arg)))
+        else:
+            argsp.append(shlex.quote(arg))
+    printc('cyan', ' '.join(argsp))
```

Comment:
(You can ignore this if this is for readability). `printc` already does `' '.join(args)`...

```python
printc('cyan', *argsp)
```